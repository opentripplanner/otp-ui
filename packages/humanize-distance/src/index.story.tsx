import React, { ReactElement } from "react";

import { Distance } from ".";

export default {
  component: Distance,
  title: "Distance"
};

export const metersShort = (): ReactElement => <Distance meters={124.5} />;

export const metersLong = (): ReactElement => <Distance long meters={124.5} />;

export const feetShort = (): ReactElement => (
  <Distance imperial meters={124.5} />
);

export const feetLong = (): ReactElement => (
  <Distance imperial long meters={124.5} />
);

export const kilometersShort = (): ReactElement => <Distance meters={12450} />;

export const kilometersLong = (): ReactElement => (
  <Distance long meters={12450} />
);

export const milesShort = (): ReactElement => (
  <Distance imperial meters={12450} />
);

export const milesLong = (): ReactElement => (
  <Distance imperial long meters={12450} />
);

export const milesFractionShort = (): ReactElement => (
  <Distance imperial meters={1245} />
);

export const milesFractionLong = (): ReactElement => (
  <Distance imperial long meters={1245} />
);
