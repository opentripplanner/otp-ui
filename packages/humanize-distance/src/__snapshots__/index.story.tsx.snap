// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Distance feetLong smoke-test 1`] = `"408 feet"`;

exports[`Distance feetShort smoke-test 1`] = `"408 ft"`;

exports[`Distance kilometersLong smoke-test 1`] = `"12.5 kilometers"`;

exports[`Distance kilometersShort smoke-test 1`] = `"12.5 km"`;

exports[`Distance metersLong smoke-test 1`] = `"125 meters"`;

exports[`Distance metersShort smoke-test 1`] = `"125 m"`;

exports[`Distance milesFraction smoke-test 1`] = `"0.8 mi"`;

exports[`Distance milesLong smoke-test 1`] = `"7.7 miles"`;

exports[`Distance milesShort smoke-test 1`] = `"7.7 mi"`;
